dlang/plugin-impl/build.gradle.kts

```diff
@@ -1,156 +1,55 @@
-                "--add-opens=java.desktop/sun.awt.image=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.awt.X11=ALL-UNNAMED", // Linux only
-                "--add-opens=java.desktop/sun.font=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.swing=ALL-UNNAMED",
-                "--add-opens=java.desktop/java.awt=ALL-UNNAMED",
-                "--add-opens=java.desktop/java.awt.peer=ALL-UNNAMED",
-                "--add-opens=java.desktop/java.awt.event=ALL-UNNAMED",
-                "--add-opens=java.desktop/javax.swing=ALL-UNNAMED",
-                "--add-opens=java.desktop/javax.swing.text.html=ALL-UNNAMED",
-                "--add-opens=java.desktop/javax.swing.plaf.basic=ALL-UNNAMED",
-                "--add-opens=java.desktop/com.sun.java.swing.plaf.gtk=ALL-UNNAMED", // Linux only
-                "--add-opens=java.desktop/com.apple.eawt.event=ALL-UNNAMED",  // Mac OSX only
-            )
-            enabled = false // workaround for https://youtrack.jetbrains.com/issue/KT-42837
+    pluginConfiguration {
+        version = properties("pluginVersion")
+        ideaVersion {
+            sinceBuild = properties("pluginSinceBuild")
+            untilBuild = properties("pluginUntilBuild")
         }
-
-        runIde {
-            enabled = true
-            jvmArgs = listOf(
-                "--illegal-access=warn",
-                "--add-opens=java.base/java.lang=ALL-UNNAMED",
-                "--add-opens=java.base/java.nio=ALL-UNNAMED",
-                "--add-opens=java.base/java.util=ALL-UNNAMED",
-                "--add-opens=java.base/sun.nio.ch=ALL-UNNAMED",
-                "--add-opens=java.base/jdk.internal.vm=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.awt=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.awt.image=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.awt.X11=ALL-UNNAMED", // Linux only
-                "--add-opens=java.desktop/sun.font=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.swing=ALL-UNNAMED",
-                "--add-opens=java.desktop/sun.java2d=ALL-UNNAMED",
-                "--add-opens=java.desktop/java.awt=ALL-UNNAMED",
-                "--add-opens=java.desktop/java.awt.peer=ALL-UNNAMED",
-                "--add-opens=java.desktop/java.awt.event=ALL-UNNAMED",
-                "--add-opens=java.desktop/javax.swing=ALL-UNNAMED",
-                "--add-opens=java.desktop/javax.swing.text.html=ALL-UNNAMED",
-                "--add-opens=java.desktop/javax.swing.plaf.basic=ALL-UNNAMED",
-                "--add-opens=java.desktop/com.sun.java.swing.plaf.gtk=ALL-UNNAMED", // Linux only
-                "--add-opens=java.desktop/com.apple.eawt.event=ALL-UNNAMED",  // Mac OSX only
-            )
-            systemProperty("idea.debug.mode", "true")
-            // The package names have to be prefixed with a #
-            systemProperty("idea.log.debug.categories", "#io.github.intellij.dlanguage")
```

Comment:
please make sure logs from our plugin are still getting logged at debug level with this removed. From what I remember it defaults to info and it's useful to see the additional output when running locally